dump/plugins/vlan.py

```diff
@@ -0,0 +1,72 @@
+from .executor import Executor
+from dump.match_infra import MatchEngine, MatchRequest
+from dump.helper import create_template_dict
+
+class Vlan(Executor):
+    
+    ARG_NAME = "vlan_name"
+    
+    def __init__(self, match_engine=None):
+        super().__init__(match_engine)
+        self.ret_temp = {}
+        self.ns = ''
+          
+    def get_all_args(self, ns=""):
+        req = MatchRequest(db="CONFIG_DB", table="VLAN", key_pattern="*", ns=ns)
+        ret = self.match_engine.fetch(req)
+        all_vlans = ret["keys"]
+        return [key.split("|")[-1] for key in all_vlans]
+            
+    def execute(self, params_dict):
+        self.ret_temp = create_template_dict(dbs=["CONFIG_DB", "APPL_DB", "ASIC_DB", "STATE_DB"])
+        vlan_name = params_dict[Vlan.ARG_NAME]
+        self.ns = params_dict["namespace"]
+        self.init_vlan_config_info(vlan_name)
+        self.init_vlan_appl_info(vlan_name)
+        self.init_state_vlan_info(vlan_name)
+        req, vlan_table = self.init_asic_vlan_info(vlan_name)
```

Comment:
Doesn't have to read the return values, as there is no dependent method's after this operation